packages/builder/plugin-swc/tests/fixtures/compat/cjs-no-preserve-dyn-import/expected.js

```diff
@@ -1,7 +1,3 @@
-"use strict";
-var _interop_require_wildcard = require("<SWC_HELPER>/_/_interop_require_wildcard");
 var a = require("foo");
 console.log(a);
-Promise.resolve().then(function() {
-    return _interop_require_wildcard._(require("other"));
-}) /*#__PURE__*/ ;
+import("other");
```

Comment:
The test case name is `cjs-no-preserve-dyn-import/expected.js`, but actually the dynamic import is prevered, should we update the test case name?

Reply:
Removed this test case as SWC only preserve dynamic import if source code is ES-module, in the old versions, we parse every file as esm, now we let SWC decide, this test code is considered script not esm, so dyn-import will be preserved